packages/api/src/endpoints/channel/HelixAdSchedule.ts

```diff
@@ -17,17 +17,17 @@ export class HelixAdSchedule extends DataObject<HelixAdScheduleData> {
 	}
 
 	/**
-	 * The UTC Unix Epoch timestamp when the broadcaster will gain an additional snooze.
+	 * The date and time when the broadcaster will gain an additional snooze.
 	 */
-	get snoozeRefreshAt(): number {
-		return this[rawDataSymbol].snooze_refresh_at;
+	get snoozeRefreshAt(): Date {
```

Comment:
Thanks for changing the return type, but it still doesn't adhere to the mentioned naming convention.